daprdocs/content/en/reference/api/distributed_lock_api.md

```diff
@@ -0,0 +1,150 @@
+---
+type: docs
+title: "Distributed Lock API reference"
+linkTitle: "Distributed Lock API"
+description: "Detailed documentation on the distributed lock API"
+weight: 660
+---
+
+## Lock
+
+This endpoint lets you acquire a lock.
+
+### HTTP Request
+
+```
+POST http://localhost:<daprPort>/v1.0-alpha1/lock/<storename>
+```
+
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+`daprPort` | The Dapr port
+`storename` | The `metadata.name` field component file. Refer to the [component schema] ({{< ref component-schema.md>}})
```

Comment:
Is store name correct here? Will locks always be implemented by a store? Or should this be called <lockcomponent>?

Reply:
You will always need a lock store. This shouldn't be called out here (and it's not called out anywhere on any other API reference either)